### DISCUSSION TITLE
Audio Firmware 1.5.15

### DISCUSSION THREAD

**Discussion by Richard153-WEB** [FrSkyRC/ETHOS-Feedback-Community#4506] [General]
MacOS Somona 14.6.1

FrSKY X20S

Hello

I was Firmware 1.4.7 Internal module 1.4.0 - 2.2.6 everything was fine, no problem.

I wanted to upgrade to Firmware 1.5.12, everything is OK, no problem, except in audio where an announcement has disappeared and it is impossible to put it back.

To try to solve the problem, I did a new firmware update to .5.1.15, internal module 1.4.0
The result is identical with the same audio problem.

Description:

The 3-position SD switch works with two announcements, except the third SD- position which remains silent.

Action:
Check my files in ‘’ Files ‘’ ‘’ audio ‘’, everything is OK and the announcement that has disappeared from SD- works very well.

All system announcements work very well too

I went to get a new audio file that is causing problems in the open TX database, the result is the same it does not work, even after going through ETHOS
Audio

In ‘’Functions‘’ ‘’SF12’’ ‘’Play audio’’ ‘’State ON’’ ‘’Condition’’ ‘’SD-’’ and at: ‘’Voice’’ it is inactive, impossible to access the announcement selection line?

Do you have a solution?

Thanks

Richard
Frenchman who has a poor command of English

**Comment by TjerkJ** [comment]
I have almost the same problems with audio.n also after update to 1.5.15
Tjerk Joustra

**Comment by Richard153-WEB** [comment]
Hello
I contacted them and no answer!
The software still has bugs.
I went back to 1.4.7 and everything is fine. It was not easy with several
hours of work.
Above all, do not upgrade for now.
Richard

**Reply by bsongis** [reply]
> Hello I contacted them and no answer! The software still has bugs. I went back to 1.4.7 and everything is fine. It was not easy with several hours of work. Above all, do not upgrade for now. Richard Le ven. 13 sept. 2024 à 22:06, TjerkJ ***@***.***> a écrit :
> […](#)
> I have almost the same problems with audio.n also after update to 1.5.15 Tjerk Joustra — Reply to this email directly, view it on GitHub <[#4506 (comment)](https://github.com/FrSkyRC/ETHOS-Feedback-Community/discussions/4506#discussioncomment-10639126)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BLGIC3GIZY7MWEERQDOZS2LZWL5PRAVCNFSM6AAAAABN76F3NSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANRTHEYTENQ> . You are receiving this because you authored the thread.Message ID: <FrSkyRC/ETHOS-Feedback-Community/repo-discussions/4506/comments/10639126@ github.com>

Perhaps it is not kind for the developers to write this when you obviously haven't taken time to read the doc?

**Comment by MIKE-DAILEY** [comment]
ETHOS 1.5.XX. has Multi Voice and there are a few things here that will help you configure it so you have voice prompts. https://www.rcgroups.com/forums/showpost.php?p=50718883&postcount=23264

**Comment by robthomson** [comment]
Agree.  Likely you have simply not selected an active voice pack to use

**Comment by Richard153-WEB** [comment]
Thank you for your answer
The problem is not there, it is in:
special functions,
for example, sf14
edit,
files, and there the box is not active, hence impossible to make a choice
of voice;

with firmware 1.4.7, internal module 1.4.0 no more problem, everything works

MacOS Somona 14.6.1

Le sam. 14 sept. 2024 à 20:02, Jet_Flyer ***@***.***> a
écrit :

**Comment by robthomson** [comment]
Are you aware that the folder structure has changed and where you store your audio files for 1.4 is different to 1.5.

**Comment by Richard153-WEB** [comment]
No, first news of this change


Le sam. 14 sept. 2024 à 21:36, Rob Thomson ***@***.***> a
écrit :

**Comment by robthomson** [comment]
[ETHOS 1.5.0 Fix Audio Callouts (1).pdf](https://github.com/user-attachments/files/17002860/ETHOS.1.5.0.Fix.Audio.Callouts.1.pdf)

**Comment by Richard153-WEB** [comment]
Thanks I will work on this subject but in MacOS
I will inform you in a few days
Richard

Le sam. 14 sept. 2024 à 21:49, Rob Thomson ***@***.***> a
écrit :
